include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -1758,7 +1755,7 @@ class FEEvaluationAccess<1, 1, Number, is_face, VectorizedArrayType>
  *         // Need to evaluate function for each component in VectorizedArray
  *         VectorizedArray<double> f_value;
  *         for (unsigned int v=0;
- *              v<VectorizedArray<double>::n_array_elements;
+ *              v<VectorizedArray<double>::size();
```

Comment:
I think this for clause would fit in one line now.

Reply:
Jup.